.github/workflows/scripts/countline.py

```
@@ -0,0 +1,295 @@
+            continue
+
+        # Create a file path
+        filepath = "{}{}{}".format(os.getcwd(), os.sep, filename)
+        if os.path.isfile(filepath) is True:
+            result.append(filepath)
+
+    # Return
+    return result
+
+
+def _filepaths_in_directories(directories):
```

Ruff (0.12.2)

Missing return type annotation for private function `_filepaths_in_directories`

(ANN202)